Watcher/v1/summit.yaml

```diff
@@ -67,3 +67,54 @@ rules:
   - name: ScriptQueue:1
   - name: ScriptQueue:2
   - name: WeatherStation:1
+- classname: DewPointDepression
+  configs:
+  - name: AT
+    dew_point_sensors:
+    - sal_index: 201  # see https://confluence.lsstcorp.org/display/LTS/ESS+CSC+Index+Registry
+      topics:
+      - topic_name: hx85ba
+        sensor_names:
+        - AuxTel-ESS01
+    temperature_sensors:
+    - sal_index: 1
+      topics:
+      - topic_name: temperature
+        sensor_names:
+        - sensor_name: AuxTel-ESS02
+          indices: [1, 2, 3]
+    warning_level: 3
+    critical_level: 2
+    hysteresis: 0.2
+    poll_interval: 10
+    max_data_age: 180
+- classname: Humidity
+  configs:
+  - name: AT
+    humidity_sensors:
+    - sal_index: 201  # see https://confluence.lsstcorp.org/display/LTS/ESS+CSC+Index+Registry
+      topics:
+      - topic_name: hx85ba
+        sensor_names:
+        - AuxTel-ESS01
+    warning_level: 73
+    hysteresis: 1
+    poll_interval: 10
+    max_data_age: 180
+- classname: OverTemperature
+  configs:
+  - name: MT Camera Hexapod
+    temperature_sensors:
+    - sal_index: 1  # see https://confluence.lsstcorp.org/display/LTS/ESS+CSC+Index+Registry
+      topics:
+      - topic_name: temperature
+        sensor_names:
+        - sensor_name: CamHex_CCW-ESS01
+          indices: [0, 1, 2, 3, 4, 5]
```

Comment:
Strictly speaking this sensor outputs 8 values, which are the 6 you have configured here plus 2 for the rotator. I understand that those are not interesting for the watcher.

Reply:
As discussed in person I thought these were the indices of the hexapod struts, but in fast they are 2-7. I have updated the file and added a comment to the file explaining the choice of indices.